config.py

```diff
@@ -8,9 +8,29 @@
 
 
 SOLR_URLS=[
-  'http://ads1/solr/update',
-  'http://ads2/solr/update',
+  'http://localhost:9983/solr/BumblebeeETL/update' 
 ]
+#  'http://ads1/solr/update',
+#  'http://ads2/solr/update',
+
+
+
+SQL = {
```

Comment:
Makes sense.  I had copied the old mongo configuration which provided for default values from an inherited object.  It's a lot of bother for such a small thing.